docs/rfcs/1013-parsec-account.md

```diff
@@ -0,0 +1,376 @@
+> Meaning they would not access the devices (encrypted by the new `vault_key`).
+
+#### Account recovery
+
+```yaml
+cmd: account_recovery_start
+req:
+  email: EmailStr
+reps:
+  ok: {}
+```
+
+Similar to `account_send_email_validation_token`, the server will send an email to recover the account.
+If the email does not exist the server will still respond with OK status for the same reason as `account_send_email_validation_token`.
+
+```yaml
+cmd: account_recovery_finish
+req:
+  recovery_token: RecoveryToken
+  auth_method_key_paramsS: KeyAlgorithm
+  encrypted_vault_key: EncryptedSymKey
+reps:
+  ok: {}
+  invalid_recovery_token: {}
+```
+
+The server will create a new [`Vault`] & [`AuthMethod`]
+
+#### Account deletion
+
+```yaml
+cmd: account_delete
+req: {}
+reps:
+  ok: {}
+```
+
+The server will send an email with a link to confirm the suppression of the account.
+
+#### Missing elements to be defined in the future
+
+- Support for FIDO2
+- Multiple auth method support (listing, deletion)
+- MFA support
+- SSO support
+- Improved vault rotation: To share the new `vault_key` with others auth methods
+
+## Alternatives Considered
+
+N/A
+
+## Operations
+
+- The authentication service storing users devices will need to be managed and probably be deployed separately from the Parsec metadata service
+- GDPR will need to be considered for the data stored by the service. T&Cs will need to be adapted in order to mention the service and how to obtain and delete the data stored.
+
+## Security/Privacy/Compliance
+
+Consideration should be taken about the method used to save the device on the remote server.
+The service would need to identify the user so we will need to have some information about the user (email mostly).
+
+### Perform clean up when recovering devices
+
+Once a device is recovered (i.e.: Was using an old vault and re-encrypted using the current `vault_key`),
+The server should clean up older devices with matching `organization_id` and `device_id` when processing the `protected_device_upload`.
+
+## Risks
+
+- The devices are not stored securely to only allow the user to access them.
+- During [perform clean up when recovering devices](#perform-clean-up-when-recovering-devices), the user can lose access to his devices if an attacker takes control of his account and replace the protected devices with dummy data).
+
+  > To remediate this, it is up to the database administrator to restore a backup.
+
+## Remarks & open questions
+
+- The client will interact with parsec-account via `libparsec`, that way we reuse our RFC communication stack.
+- What to do when a user is revoked from an organization?
+  Should we remove the device, or should we just provide the information that the device for the given organization is revoked?
+
+  > Just inform the user about the revocation
+  >
+  > @touilleMan
+
+- A similar question applies for devices that are no longer used (e.g. a linked browser that is no longer used)
```

Comment:
This could also apply to a old laptop's device.  But I think it's more a parsec metadata issue of having the option to remove unused devices (and then propagating this data to parsec account)